src/room/MuteStates.ts

```diff
@@ -75,7 +75,7 @@ export function useMuteStates(): MuteStates {
 
   const { skipLobby } = useUrlParams();
   // In SPA without lobby we need to protect from unmuted joins (Privacy).
-  const allowStartUnmuted = !skipLobby || !!widget;
+  const allowStartUnmuted = !skipLobby || !widget;
```

Comment:
This does not look right->
 - in SPA - `widget = undefined/null`
 - allowStartUnmuted will always be `true` (because !widget == true)
 - but we want it to be false in the spa.

Reply:
Yes, I already reverted it.

Reply:
I saw it too late sorry.